mscore/musescore.cpp

```diff
@@ -4948,13 +4955,28 @@ int main(int argc, char* av[])
             QString temp = parser.value("T");
             if (temp.isEmpty())
                    parser.showHelp(EXIT_FAILURE);
-            trimMargin = temp.toInt(); // ToDo: check whether conversion went OK, set to -1 if not
+            bool ok = false;
+            trimMargin = temp.toInt(&ok);
+            if (!ok)
+                  trimMargin = -1;
            }
       if (parser.isSet("x")) {
             QString temp = parser.value("x");
             if (temp.isEmpty())
                    parser.showHelp(EXIT_FAILURE);
-            guiScaling = temp.toDouble(); // ToDo: check whether conversion went OK, set to 1.0 if not
+            bool ok = false;
+            guiScaling = temp.toDouble(&ok);
+            if (!ok)
+                  guiScaling = 0.0;
```

Comment:
Maybe 1.0 would be better no?

Reply:
I have a computer that breaks - I don't know if you can duplicate the problem remotely, but I'm willing to give you access if that would fix this.

Using -x 1.0 fixed the problem for me.

It is a "TV" monitor that has a VGA connector on it, so the theory about something being funny in the hardware seems to be a good guess.

Have anything you want me to debug about it?